Header header
float64 zTildeHomog
float64 zTildeMocap
float64 zTildePercentHomog
float64 zTildePercentMocap
float64 zStar
float64 zHatHomog
float64 zHatMocap
float64 zHatDotHomog
float64 zHatDotMocap
